orangecontrib/text/preprocess.py

```diff
@@ -1,281 +0,0 @@
-import nltk
-from nltk.corpus import stopwords
-from nltk.stem import WordNetLemmatizer
-from nltk.stem.porter import PorterStemmer
-from nltk.stem.snowball import SnowballStemmer
-from nltk.tokenize import word_tokenize, RegexpTokenizer, TweetTokenizer
-
-from orangecontrib.text.corpus import Corpus
-
-# Ensure NLTK data is present
-NLTK_DATA = ["wordnet",  "stopwords",  "punkt"]
```

Comment:
Did you put this somewhere else? I agree this is a bit hackish but we need this somewhere until we figure out a better solution. Without it required NLTK data is not downloaded and preprocessing does not work. Consequently most text widgets also break.

I cleared caches on Travis and rerun the tests to show where is the problem.